bugzilla2gitlab/models.py

```diff
@@ -44,17 +44,16 @@ def save(self):
             comment.save()
 
         # close the issue in GitLab, if it is resolved in Bugzilla
-        if self.issue.status == "RESOLVED":
+        if self.issue.status in ["RESOLVED", "VERIFIED", "CLOSED"]:
             self.issue.close()
 
 
 class Issue(object):
     '''
     The issue model
     '''
-    required_fields = ["sudo", "title", "description", "status"]
-    data_fields = ["sudo", "title", "description", "status", "assignee_id", "milestone",
-                   "labels"]
+    required_fields = ["title", "description"]
```

Comment:
Here and other places you remove references to `sudo`. It doesn't look like `sudo` has been deprecated in the v4 API. Why did you remove it?